### PR TITLE
Feature/offline upgrade

### PR DESCRIPTION
**Pull request by seriva**
- #640 
- #644

### CODE REVIEW

File: core/src/epicli/cli/helpers/build_saver.py [hunk 1]
```diff
@@ -10,8 +10,11 @@
 MANIFEST_FILE_NAME = 'manifest.yml'
 SP_FILE_NAME = 'sp.yml'
 INVENTORY_FILE_NAME = 'inventory'
+INVENTORY_FILE_NAME_LEGACY = 'inventory/development'
```
Comment (toszo) [inline core/src/epicli/cli/helpers/build_saver.py]: For legacy builds this path will not always be like that. Possible is: inventosy/staging, inventory/production, or other (was defined by user)

Comment (seriva) [reply core/src/epicli/cli/helpers/build_saver.py]: But the it will be in the folder inventory/?

Comment (erzetpe) [reply core/src/epicli/cli/helpers/build_saver.py]: This should be always in the folder inventory, but names can differ.

Comment (seriva) [reply core/src/epicli/cli/helpers/build_saver.py]: Fixed

**Pull request merged**